### ISSUE TITLE
An installation error occured

### ISSUE THREAD

**Issue by 6axis** [HALRobotics/Beta#113]
Hello, 

It seem that I'm unable to update/re-install/install HAL on my computer. The installation process go well until this message pop-up : 
![noluck](https://user-images.githubusercontent.com/52275746/133135429-fea3b9d4-20a8-4479-a445-c3bb26850a66.JPG)

I unistalled the framework, installed it with a freshly downloaded setup, installed only the application with no extentions, and with all the extensions... no luck for now. 

Do you have any advices to do a deeper uninstall of HAL to (maybe) solve the issue ?

I had no problem doing the last update one my other computer and never had problem before using HAL on the one that is problematic today.
Rhino is in 7.9.21222.1500 official licence
W10 is 21H1. 

Best regard

**Comment by sebandraos** [comment]
Hi @6axis , If you could send us over the installation log (C:\ProgramData\HAL\Installation.log) then hopefully we can see what's causing the installation to fail. Thanks.

**Comment by 6axis** [comment]
Hi @sebandraos, here it is : 

[Installation.log](https://github.com/HALRobotics/Beta/files/7156582/Installation.log)

Thanks for your help !

**Comment by sebandraos** [comment]
There are a lot of errors like - `L'accès au chemin d'accès 'C:\ProgramData\HAL\HAL.Communication.dll' est refusé.` This sort of file access issue normally means that an installation was once run As Admin and now is not. Did you manage to install the update without extensions?
One option is to attempt to install as admin again but given that that's unnecessary to in the HAL Robotics Framework, I would suggest completely uninstalling (including the installer itself), then deleting:
`C:\ProgramData\HAL` and `%localappdata%\HAL` before reinstalling again from scratch.
Let us know how you get on.

**Comment by 6axis** [comment]
Ok ! it worked !

Yes I tried to install it as an admin in a desperate attempt to make the error disapear... But before that I didn't manage to install only the application without extensions.

So I deleted the two folders you said and reinstalled it normaly and the problem is now gone. Maybe the log is more explicit than me, it's way over my understanding ans skills !

Thank you so much for your help ! Hope it can help somehow.
